static/app/components/performance/waterfall/constants.tsx

```diff
@@ -1,2 +1,15 @@
 export const ROW_HEIGHT = 24;
 export const ROW_PADDING = 4;
+
+export enum SpanBarHatch {
+  gap = 'gap',
+  affected = 'affected',
+}
+
+export const SPAN_HATCH_TYPE_COLOURS: Record<
+  SpanBarHatch,
+  {alternate: string; primary: string}
+> = {
+  [SpanBarHatch.gap]: {primary: '#dedae3', alternate: '#f4f2f7'},
+  [SpanBarHatch.affected]: {primary: '#F55459', alternate: '#FAA9AC'},
```

Comment:
Are these colours net new? Are they part of the theme? If so they should either be pulled from the color consts or we should move these hexs there.

Reply:
I tried passing in the theme obj and getting the colours from there, but our theme doesn't have all of the require colours available. The colours from the gap spans were also hardcoded previously